### PR TITLE
Chore: configure prettier

### PR DESCRIPTION
**Pull request by pepopowitz**
Collaboration with @annacarey.

It's time for this project to put on its grown-up pants and enforce formatting, to avoid noisy churn in PRs. 

- Installs prettier as a dependency
- Runs prettier against all existing files
- Configures prettier to run from a pre-commit hook via husky (though @annacarey and I found that with my flow of using `git commit -a` instead of `git add ... git commit`, the hook gets skipped). 

I still want to take a good thorough look through the diff here to make sure it didn't mess anything up. Also, @annacarey is going to confirm for me that this branch lives up to my claims when she pulls it down.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,10 +36,31 @@
...
     "typescript": "^4.1.2"
+  },
+  "prettier": {
+    "arrowParens": "avoid",
+    "bracketSpacing": true,
+    "printWidth": 80,
+    "semi": false,
+    "singleQuote": false,
+    "trailingComma": "es5"
+  },
```
Comment (pepopowitz) [inline package.json:55]: We just copied these settings from force.

File: package.json [hunk 2]
```diff
@@ -15,6 +15,8 @@
     "relay": "^0.8.0-1"
   },
   "scripts": {
+    "prepare": "husky install",
```
Comment (pepopowitz) [inline package.json:18]: I _think_ this probably copies what's configured in the package.json into a pure git hook? Not totally sure, but it's [in the setup docs](https://typicode.github.io/husky/#/?id=install).

File: src/exercises/01-Query-Renderer/README.md [hunk 3]
```diff
@@ -245,53 +254,56 @@ Arguments can be passed into the `query` of a `QueryRenderer` through the `varia
...
 
 ```typescript
-    <QueryRenderer<Artist1QueryRendererQuery>
-      // ...
-      variables={{artistID: '1'}}
-      // ...
-    />
+<QueryRenderer<Artist1QueryRendererQuery>
+  // ...
+  variables={{ artistID: "1" }}
```
Comment (pepopowitz) [inline src/exercises/01-Query-Renderer/README.md:281]: This is a nice fix to a code-block inside a .md, but I plan to take an extra scrutinous eye to the rest of the changes to make sure it doesn't do anything unexpected in the existing exercise instructions.

**Review by pepopowitz — COMMENTED** [review]
Comments for context

**Review by olerichter00 — APPROVED** [review]
Looks good to me 🎉  I pulled the changes and everything looks fine and seems to work as expected.

**Pull request merged**